example/thirdparty/ollama-js/build.mill

```diff
@@ -0,0 +1,362 @@
+//// The Ollama JavaScript library provides the easiest way to integrate
+//// your JavaScript project with Ollama. https://github.com/ollama/ollama.
+//
+//// Project home: https://github.com/ollama/ollama-js
+
+//// Project Layout:
```

Comment:
This description is useless. The point is to describe what this example is with regard to Mill, not in isolation